### PR TITLE
Add Norwegian translation

### PR DESCRIPTION
**Pull request by RainoBoy97**
#1 Add Norwegian translation

### CODE REVIEW

File: lang.nb.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*------------------
+Language: English
```
Comment (miyurusankalpa) [inline lang.nb.php]: can you change the comment. thanks

**Comment by RainoBoy97** [comment]
Oops, forgot that! Fixed now 😄

**Comment by miyurusankalpa** [comment]
merged, thanks 👍
have a great hacktoberfest 😃

**Comment by RainoBoy97** [comment]
No problem! Happy hacktoberfest 😸

**Pull request merged**